scannerjenkins.properties

```
@@ -6,8 +6,13 @@ organizationUrl=https://www.sonarsource.com
 homepageUrl=https://docs.sonarqube.org/latest/analysis/scan/sonarscanner-for-jenkins/
 issueTrackerUrl=https://jira.sonarsource.com/projects/SONARJNKNS/issues
 sourcesUrl=https://github.com/SonarSource/sonar-scanner-jenkins
-archivedVersions=2.10,2.11,2.12,2.13.1
-publicVersions=2.14
+archivedVersions=2.10,2.11,2.12,2.13.1,2.14
+publicVersions=2.15
+
+2.15.description=Bug fix (Allow restricting search for task report file)
```

The explanation doesn't seem to correspond to the bug that was fixed: SonarQube Jenkins plugin keeps creating threads when querying a deleted project.
Seems to come from a 2.16 ticket: [SONARJNKNS-355](https://sonarsource.atlassian.net/browse/SONARJNKNS-355)
```suggestion
2.15.description=Out of memory bug fix
```

---

how am I supposed to understand that when this is the ticket linked in the REL ticket... 😮‍💨

---

Thanks for catching this btw ;-)